src/transformers/models/bart/modeling_tf_bart.py

```diff
@@ -1018,8 +1038,7 @@ def get_output_embeddings(self):
     "The BART Model with a language modeling head. Can be used for summarization.",
     BART_START_DOCSTRING,
 )
-class TFBartForConditionalGeneration(TFPretrainedBartModel):
-    base_model_prefix = "model"
+class TFBartForConditionalGeneration(TFPretrainedBartModel, TFCausalLanguageModelingLoss):
```

Comment:
Can we please not change the loss computation in this PR. `TFCausalLanguageModelingLoss` is different from Bart's compute loss function because Bart's loss function explicitly uses the pad_token_id as the ignore token and not -100. Sam did some special Bart things here that I'd like to take a look at in another PR.

Reply:
Ok, I put back the way as it was to compute the loss.